### PR TITLE
EVI Web Example Polishing

### PR DESCRIPTION
**Pull request by norman-codes**
Minor details on the EVI web examples using TypeScript, the NextJS Pages Router, and the NextJS App Router were changed.
- An example `.env` file (`.env.example`) was added to the TypeScript example and references to it (and using a `.env` file for authentication) were updated in the corresponding README. Note that it was attempted to have the placeholder file be named `.env` directly, but after researching best practices for environment variable files in GitHub repositories, I decided to use `.env.example` instead.
- The name of the export function for the Messages.tsx components in the EVI NextJS App Router and Pages Router example repositories is changed to match the component's name.

### CODE REVIEW

File: evi-typescript-example/README.md [hunk 1]
```diff
@@ -48,7 +48,10 @@ export VITE_HUME_API_KEY=<YOUR_API_KEY>
 export VITE_HUME_CLIENT_SECRET=<YOUR_CLIENT_SECRET>
 ```
 
-You can make these environment variables persistent by adding them to the `.env` file in the root folder of the repo, which currently exists with placeholders. Once the API secrets have been added to the file, they will persist there and you do not have to set them as environment variables going forward. The `.env` file is listed in the `.gitignore` of the repository, so there is no risk of accidentally committing these secrets to a public repository. 
+You can make these environment variables persistent by adding them to a file named `.env` in the root folder of the repo.
+> There is an example file called `.env.example` with placeholder values, which you can simply rename to `.env`.
+
+Once the API secrets have been added to the `.env` file, they will persist there and you do not have to set them as environment variables going forward. Importantly, the `.env` file is listed in the `.gitignore` of the repository, so there is no risk of accidentally committing these secrets to a public repository.
```
Comment (zgreathouse) [inline evi-typescript-example/README.md:54]: I think we can remove this paragraph. 

The assurance that the `.env` file is listed in the `.gitignore` is unnecessary. (This assurance would only benefit developers contributing to this repo— which will be only members of the Hume team.)

File: evi-typescript-example/README.md [hunk 2]
```diff
@@ -48,7 +48,10 @@ export VITE_HUME_API_KEY=<YOUR_API_KEY>
 export VITE_HUME_CLIENT_SECRET=<YOUR_CLIENT_SECRET>
 ```
 
-You can make these environment variables persistent by adding them to the `.env` file in the root folder of the repo, which currently exists with placeholders. Once the API secrets have been added to the file, they will persist there and you do not have to set them as environment variables going forward. The `.env` file is listed in the `.gitignore` of the repository, so there is no risk of accidentally committing these secrets to a public repository. 
+You can make these environment variables persistent by adding them to a file named `.env` in the root folder of the repo.
+> There is an example file called `.env.example` with placeholder values, which you can simply rename to `.env`.
```
Comment (zgreathouse) [inline evi-typescript-example/README.md:52]: Suggested edit:

> Refer to the [.env.example](<url-to-example-file>) file for which environment variables to include in your `.env` file.

File: evi-typescript-example/README.md (hunk 1 above)
Comment (norman-codes) [reply evi-typescript-example/README.md:54]: Sounds good and agreed!

File: evi-typescript-example/README.md (hunk 2 above)
Comment (norman-codes) [reply evi-typescript-example/README.md:52]: Thank you; agreed - I'll make that edit now and follow up with its commit.

**Review by zgreathouse — APPROVED** [review]

**Pull request merged**